### PR TITLE
Bump version, update release notes

### PR DESCRIPTION
**Pull request by mklewandowski**
Bump the version number and update the changelog in preparation for version 2..3.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,56 @@
...
+- SEISMIC: Update button styling [177430008](https://www.pivotaltracker.com/story/show/177430008)
+- SEISMIC: Plate Colors in Deformation Sim [177208666](https://www.pivotaltracker.com/story/show/177208666)
+- SEISMIC: Set Point Tool [177256960](#177256960)
+- SEISMIC: Plate Motion Widgets in Deformation Sim [174846688](https://www.pivotaltracker.com/story/show/174846688)
+- SEISMIC: GPS Location vs Time Graph Styling [175037328](https://www.pivotaltracker.com/story/show/175037328)
+- SEISMIC: Explore Direction Widget [176787693](https://www.pivotaltracker.com/story/show/176787693)
+- SEISMIC: Make Explore Direction Widget Movable [177429998](https://www.pivotaltracker.com/story/show/177429998)
+- TEPHRA: Make tephra distribution colors semi-transparent [169984469](https://www.pivotaltracker.com/story/show/169984469)
+
 ## Version 2.2.3 - release January 20, 2020
```
Comment (kswenson) [inline CHANGELOG.md:40]:
```suggestion
## Version 2.2.3 - release January 20, 2021
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,56 @@
 # Changelog
 
+## Version 2.2.3 - release March 25, 2021
+
+### Bug Fixes
```
Comment (kswenson) [inline CHANGELOG.md:5]: I argued elsewhere, and received at least some agreement, that `Features/Improvements` should come before `Bug Fixes`. Seems pretty obvious to me, but.... 😉

**Review by kswenson — APPROVED** [review]
👍 Just a couple of minor suggestions.

File: CHANGELOG.md (hunk 1 above)
Comment (mklewandowski) [reply CHANGELOG.md]: Ah, thanks!  A bunch of these had the wrong date.  Looks like I missed one.

File: CHANGELOG.md (hunk 2 above)
Comment (mklewandowski) [reply CHANGELOG.md]: Sounds good to me!  Fixed.

**Pull request merged**